master/buildbot/status/web/builder.py

```diff
@@ -602,10 +602,13 @@ def content(self, req, cxt):
         status = self.getStatus(req)
         encoding = getRequestCharset(req)
 
-        showCategories = req.args.get("category", [])
-        if showCategories == []:
-            showCategories = None
-        builders = req.args.get("builder", status.getBuilderNames(categories=showCategories))
+        showTags = req.args.get("tag", [])
+        if not showTags:
+            showTags = req.args.get("category", [])
+            if not showTags:
+                showTags = None
+
+        builders = req.args.get("builder", status.getBuilderNames(tags=showTags))
```

Comment:
For the status objects, I decided it was probably better to not break existing bookmarks, so all the request checks will accept 'category' (if 'tag' wasnt there).